clarity/src/vm/tests/traits.rs

```diff
@@ -66,7 +68,7 @@ fn test_all() {
         test_let3_trait,
     ];
     for test in to_test.iter() {
-        with_memory_environment(test, false);
+        with_memory_environment(test, StacksEpochId::latest(), false);
```

Comment:
Should this be tested in all epochs?

Reply:
This one will be a bit trickier, because some of these tests only work in 2.1. I'll have to go through and split them up into those that can run on both 2.05 and 2.1, and those that can only run on 2.1 (because they rely on Clarity2).